lib/Agrammon/DB/Dataset.pm6

```diff
@@ -147,31 +163,51 @@ class Agrammon::DB::Dataset does Agrammon::DB {
         return self;
     }
 
-    method set-tag($tagName) {
+
+    #| Set tag on datasets.
+    method set-tag(@datasets!, $tag-name! --> Nil) {
```

Comment:
Redundant `!` (positional parameter are required by default). Funnily enough, we introduced an annotation to point that out in this week's Comma release. :-)

Reply:
Removed.